Update mount parameters (#4582)
ArduPilot/ardupilot_wiki: common/source/docs/common-gremsy-pixyu-gimbal.rst

> Connect with a ground station and set the following parameters:

Reviewer: ```suggestion
Connect with a ground station and set the following parameters, if using the first mount:
```

- :ref:`MNT1_TYPE <MNT1_TYPE>` to "4" for "SToRM32 MavLink" and reboot the autopilot
- :ref:`SERIAL2_BAUD <SERIAL2_BAUD>` to "115" for 115200 bps.  "SERIAL2" can be replaced with another serial port (i.e. SERIAL1) depending upon the physical connection
- :ref:`SERIAL2_PROTOCOL <SERIAL2_PROTOCOL>` to 2 for "MAVLink2"
- :ref:`SR2_EXTRA1 <SR2_EXTRA1>` to 10
- :ref:`SR2_POSITION <SR2_POSITION>` to 10

> The gimbal's maximum angles can be set using these parameters:

Reviewer: ```suggestion
The gimbal's maximum angles can be set using these parameters (shown for the first mount):
```

- :ref:`MNT1_ROLL_MIN <MNT1_ROLL_MIN>` to -30 to allow leaning left up to 30deg
- :ref:`MNT1_ROLL_MAX <MNT1_ROLL_MAX>` to 30 to allow leaning right up to 30deg
- :ref:`MNT1_PITCH_MIN <MNT1_PITCH_MIN>` to -90 to allow pointing 90deg down
- :ref:`MNT1_PITCH_MAX <MNT1_PITCH_MAX>` to 30 to allow pointing 30deg up
- :ref:`MNT1_YAW_MIN <MNT1_YAW_MIN>` to -180 to allow turning around to the left
- :ref:`MNT1_YAW_MAX <MNT1_YAW_MAX>` to 180 to allow turning around to the right

> To control the gimbal's lean angles from a transmitter set :ref:`RCx_OPTION <RC1_OPTION>` to the respective control axis.

Reviewer: ```suggestion
To control the gimbal's lean angles from a transmitter set the RC controls for roll, tilt, or pan using ``RCx_OPTION`` 212 (Mount1 Roll), 213 (Mount1 Pitch), 214 (Mount1 Yaw) for the first mount, or 215-217 for the second mount, respectively.
```

> Connect with a ground station and set the following parameters:

Reviewer: ```suggestion
Connect with a ground station and set the following parameters, if using the first mount::
```
